include/SFML/Window/Event.hpp

```diff
@@ -399,7 +400,16 @@ class Event
     }
 
     template <typename T>
-    static constexpr bool isEventSubtype = isInParameterPack<T>(decltype (&m_data)(nullptr));
+    static constexpr bool isEventSubtype = isInParameterPack<T>(static_cast<decltype(&m_data)>(nullptr));
+
+    template <typename Handler, typename... Ts>
+    static constexpr bool isInvokableWithAnyOf(std::variant<Ts...>*)
+    {
+        return (std::is_invocable_v<Handler, Ts&> || ...);
```

Comment:
```suggestion
        return std::disjunction_v<std::is_invocable<Handler, Ts&>...>;
```
Could we do something like this instead?

Reply:
Yes, this feels more modern